File: beecrowd/beginner/c/1048.c
#include <stdio.h>
#define INF 3e38

int main() {
    float salary, readjustment, new_salary; 
    const float SALARY_CAP[5] = {400.0, 800.0, 1200.0, 2000.0, INF};
    const float READJUSTMENT_RATE[5] = {0.15, 0.12, 0.10, 0.07, 0.04};

    scanf("%f", &salary);

    for (int i = 0; i < 5; i++){
        if (salary <= SALARY_CAP[i]) {
            readjustment = READJUSTMENT_RATE[i] * salary;
            new_salary = salary + readjustment;
            printf("Novo salario: %.2f\n", new_salary);
            printf("Reajuste ganho: %.2f\n", readjustment);
            printf("Em percentual: %.0f %%\n", READJUSTMENT_RATE[i] * 100);
            break;
        }
    }
    return 0;
    
}

File: beecrowd/beginner/python/1048.py
salary = float(input())

if salary <= 400:
    percentage = 0.15

elif salary <= 800:
    percentage = 0.12

elif salary <= 1200:
    percentage = 0.10

elif salary <= 2000:
    percentage = 0.07

else:
    percentage = 0.04


adjustment = salary * percentage
new_salary = salary + adjustment

print(f'Novo salario: {new_salary:.2f}')
print(f'Reajuste ganho: {adjustment:.2f}')
print(f'Em percentual: {percentage * 100:.0f} %')
